### PR TITLE
Trim

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/compiler-documenting.md [hunk 1]
```diff
@@ -35,28 +35,18 @@ Much like individual tests or building certain components you can build only
...
-But, when enabled, compiler documentation does include internal items.
+Compiler documentation is not built by default. To enable it:
 
-Next open up config.toml and make sure these two lines are set to true:
-
-```bash
-docs = true
+```toml
+[build]
 compiler-docs = true
```
Comment (JohnTitor) [inline src/compiler-documenting.md]: It is useful to know that this flag is in `config.toml`.

Comment (tshepang) [reply src/compiler-documenting.md]: oh, indeed

File: src/compiler-documenting.md [hunk 2]
```diff
@@ -35,28 +35,18 @@ Much like individual tests or building certain components you can build only
...
 ```
 
-When you want to build the compiler docs as well run this command:
-
-```bash
-./x.py doc
```
Comment (mark-i-m) [inline src/compiler-documenting.md]: Do we have this command documented somewhere?

Comment (tshepang) [reply src/compiler-documenting.md]: it is, beginning of the document

**Comment by mark-i-m** [comment]
It looks like there is a merge conflict

**Pull request merged**